coral/pnpm-lock.yaml

```diff
@@ -1004,7 +1007,6 @@ packages:
       strip-ansi-cjs: /strip-ansi@6.0.1
       wrap-ansi: 8.1.0
       wrap-ansi-cjs: /wrap-ansi@7.0.0
-    dev: false
```

Comment:
Wonder why these dependencies are not `dev` any more :o

Reply:
aivenio/aquarium has it defined as a dependency, so that's where that's coming from. But I'm not sure why the change happened in this PR, where we didn't update aquarium 🤔  But it's from there, so it's correct.